app/controllers/chatfuel_controller.rb

```diff
@@ -7,14 +7,83 @@ def show
     if tc
       text = Text::Renderer.new(text_component: tc).render(:main_part)
 
-      @response = {
-        messages: [
-          { text: text.first(640)} # Messages for Facebook Messenger can only be 640 characters long. Source: https://developers.facebook.com/docs/messenger-platform/send-api-reference#request
-        ]
-      }
-      render json: @response
+      qa = tc.question_answers
+
+      #Test API Request: http://localhost:3000/chatfuel/1
+      if qa.exists?(1)
```

Comment:
I never use `::exists?(id)`. According to [ApiDoc](https://apidock.com/rails/ActiveRecord/Base/exists%3F/class), this will find a QuestionAnswer with `id == 1`. Why do you want to find a particular QuestionAnswer record?